CodeMaid/UI/Dialogs/CleanupProgress/CleanupProgressViewModel.cs

```diff
@@ -176,7 +176,7 @@ private void backgroundWorker_ProgressChanged(object sender, ProgressChangedEven
             dynamic currentItem = e.UserState;
 
             CountProgress = currentCount;
-            CurrentFileName = currentItem.Name;
+            CurrentFileName = string.Format(StringResourceKey.Cleaning0, currentItem.Name);
```

Comment:
This isn't the current file name anymore, but an output string.  I think this was done to avoid using the StringFormat in XAML.  If possible it would be preferable to keep the StringFormat in XAML, since this class is a ViewModel.

Reply:
I tried the XAML stringformat, but it doesn't seem to work.

Reply:
Ok, another approach would be to split the label into two components then.  One for the "Cleaning" which can be translated directly and one for the CurrentFileName.  That would allow this ViewModel to be reverted to not having View code in it.